src/main/java/net/dv8tion/jda/core/handle/VoiceStateUpdateHandler.java

```diff
@@ -144,9 +146,10 @@ else if (channel == null)
                     //If we have connected (VOICE_SERVER_UPDATE received and AudioConnection created (actual connection might still be setting up)),
                     // then we need to stop sending audioOpen/Move requests through the MainWS if the channel
                     // we have just joined / moved to is the same as the currently queued audioRequest
-                    // (handled by removeAudioConnection
+                    // (handled by updateAudioConnection)
                     if (mng.isConnected())
-                        api.getClient().removeAudioConnection(guildId, channel);
+                        api.getClient().updateAudioConnection(guildId, channel);
+                    //If we are not already connected this will be removed by VOICE_SERVER_UPDATE
```

Comment:
Technically this comment about VOICE_SERVER_UPDATE belongs in the above `if (oldChannel == null)` block